library/src/meta/numbering.rs

```diff
@@ -360,6 +363,59 @@ impl NumberingKind {
                 let symbol = SYMBOLS[(n - 1) % SYMBOLS.len()];
                 let amount = ((n - 1) / SYMBOLS.len()) + 1;
                 std::iter::repeat(symbol).take(amount).collect()
+            },
+            Self::Hebrew => {
+                if n == 0 {
+                    return '-'.into();
+                }
+
+                let mut fmt = EcoString::new();
+                'outer: for &(name, value) in &[
+                    ('ת', 400),
+                    ('ש', 300),
+                    ('ר', 200),
+                    ('ק', 100),
+                    ('צ', 90),
+                    ('פ', 80),
+                    ('ע', 70),
+                    ('ס', 60),
+                    ('נ', 50),
+                    ('מ', 40),
+                    ('ל', 30),
+                    ('כ', 20),
+                    ('י', 10),
+                    ('ט', 9),
+                    ('ח', 8),
+                    ('ז', 7),
+                    ('ו', 6),
+                    ('ה', 5),
+                    ('ד', 4),
+                    ('ג', 3),
+                    ('ב', 2),
+                    ('א', 1),
+                ] {
+                    while n >= value {
+                        match n {
+                            15 => fmt.push_str("ט\"ו"),
```

Comment:
The special cases write 15, 16 as (9+6), (9+7) instead of (10+5), (10+6) in order not to spell the name of G-d. This would also apply to 115, 116 so it cannot be extracted from the loop.

Reply:
There are also other edge cases such as avoiding the possibility of negative words which should be added later, I just don’t have a full list and they are for relatively high numbers

Reply:
That might be clearer, although it would then probably duplicate 15 and 16? I'll let you decide, just tell me when its ready to merge.

Reply:
imho it's better the way it is now.